scripts/Packages/Package_BGA/bga.yaml

```diff
@@ -345,3 +345,14 @@ WLP-9_1.448x1.468mm_Layout3x3_P0.4mm_Ball0.27mm_Pad0.25mm:
   paste_margin: 0.000001
   layout_x: 3
   layout_y: 3
+Dialog_VFBGA-86_6x6mm_Layout10x10_P0.55mm_Ball0.25mm_Pad0.2mm:
+  description: "VFBGA-86"
+  size_source: "https://www.dialog-semiconductor.com/sites/default/files/da1469x_datasheet_3v1.pdf"
+  body_size_x: 6.0
+  body_size_y: 6.0
+  pitch: 0.55
+  pad_diameter: 0.205
+  mask_margin: 0.026
```

Comment:
Where did you get those values from (pad_diameter, mask_margin)? I feel stupid since I can't find it.

Reply:
It's experimental data. I've made 7 boards with this MCU since my PR

Reply:
Fair, but we try to not base footprints on experimental value if possible. The manufacturers recommendations are to be followed. But here we don't have any.

In this case I suggest to use 0.2mm for the pad as specified in IPC-7351B (https://github.com/pointhi/kicad-footprint-generator/blob/d3b8ffd00e656a0ccff3a75be2c9cb8aeec3f6df/scripts/Packages/Package_BGA/ipc_definitions.yaml)

Also, I checked the snapeda.com export (dialog links to them for footprints/symbols). They have 0.2mm as well.
```
  (pad A1 smd circle (at -2.475 -2.475) (size 0.2 0.2) (layers F.Cu F.Mask F.Paste))
  (pad A2 smd circle (at -1.925 -2.475) (size 0.2 0.2) (layers F.Cu F.Mask F.Paste))
``` 

Regarding mask margin I am actually not sure how to continue. Never had that issue before.
* https://github.com/KiCad/kicad-footprints/pull/1170#issuecomment-447583481 is about the same value
* I agree that having a value set that is proven in production is a good idea.
* The KLC does explicitly allow to set the mask in [F9.2](https://kicad-pcb.org/libraries/klc/F9.2/).
* From https://www.ti.com/lit/an/sprabb3/sprabb3.pdf?#page=10
  * ![image](https://user-images.githubusercontent.com/17516572/91091914-971c9f00-e657-11ea-97a1-ecfecc6b0ee9.png)
*![image](https://user-images.githubusercontent.com/17516572/91091952-9f74da00-e657-11ea-9927-0eb4438c9091.png)
* One option would be to remove the `mask_margin` parameter completely. Then the settings as defined in the board-file where the footprint is used will be applied to the pads.
* Or take the value from TI (0.05mm) which is 10 years old
* Or take 0.035 which lots of other parts in that file use
* Or we just go with your proven value which is close to other values in the same file. One question here though, why did you choose that value?

Reply:
Regarding your last question: it's a value, recommended by the local PCB manufacturer. Anyway, I liked your idea to take the most common value from this file (0.035)